apps/nextjs-app-router/src/app/[lang]/[[...path]]/page.tsx

```diff
@@ -3,19 +3,27 @@ import '@/makeswift/components'
 import { getSiteVersion } from '@makeswift/runtime/next/server'
 import { notFound } from 'next/navigation'
 import { Page as MakeswiftPage } from '@makeswift/runtime/next'
-import { locales } from '@/localization'
 
 type ParsedUrlQuery = { lang: string; path?: string[] }
 
 export async function generateStaticParams() {
-  const pages = await client.getPages()
 
-  return pages.flatMap((page) =>
-    locales.map((locale) => ({
-      path: page.path.split('/').filter((segment) => segment !== ''),
-      lang: locale,
+  const pages = await client.getPages().toArray()
```

Comment:
I'm ok with leaving this like this for now but I don't think we should recommend listing all pages in general. Users should be getting a subset of their pages that are important. Once we have a good example of this we should update our docs so that we're not steering users in the what I think is the wrong direction here